src/vendor.scss

```diff
@@ -9,6 +9,6 @@
 */
 @import '~patternfly-react/dist/css/patternfly-react.css';
 
-@import "~react-select/dist/react-select.css";
+/* @import "~react-select/dist/react-select.css"; */
```

Comment:
Is this relevant?
Doesn't seem related to quadicons..

Reply:
Not related to quadicons as are not several other commits above.

Only needed to make the stuff actuall work as whole.

Do you want me to separate the unrelated stuff (already in separate commits) to a separate PR?

Reply:
Nope, just wanted to make sure this is intentional and fixing things, not breaking other things :).
(More like, sounds it could break some selects?)

But I see the commit message mentions it's missing, so, probably not breaking anything.

Reply:
I did not know about his part do I just commented this out. I hope someone who knows about the selects can take a look in a separate PR.

However for now w/o commenting this out, the package would not build.